src/create-render-context.js

```diff
@@ -214,9 +209,17 @@ const createRenderContext = function(locationUrl, globals, jsPackages) {
 
     // Now we execute inside the sandbox context each script package.
     let cumulativePackageSize = 0;
-    jsPackages.forEach(script => {
-        context.runVMScript(script);
-        cumulativePackageSize += script.size;
+    jsPackages.forEach(({content, url}) => {
+        context.run(content, {filename: url});
```

Comment:
Which part of this PR changed it so that JSDOM runs the things that it loads?  I'm having a hard time picking that part out from the rest of the changes.

Reply:
The changes in create-render-context.js. That's where the custom resource loader is defined. JSDOM uses that to get the files and then uses the buffers we return to load the scripts.

Reply:
I understand now.  Thank you!